### ISSUE TITLE
Weird style of button on iOS

### ISSUE THREAD

**Issue by phuclh** [quasarframework/quasar#7864]
**Describe the bug**
I am using the quasar button and it is good on Android devices, if I use a iOS device, it will show like this

**Screenshots**
![large-PNG image](https://user-images.githubusercontent.com/6707194/95027519-ff46a380-064d-11eb-8caa-da0830f00ba6.png)

<img width="118" alt="Screen Shot 2020-10-04 at 2 31 36 PM" src="https://user-images.githubusercontent.com/6707194/95027558-577da580-064e-11eb-9772-8001937bc9a4.png">

```
<q-btn v-if="isProposalWindowOpen(item.bid_request)"
       :to="{ name: 'proposal-create', params: {request_id: item.bid_request.id} }"
       color="primary"
       width="100%"
       unelevated
       class="w-full no-shadow text-bold text-base"
       content-class="text-sm">
    <span class="text-base text-bold py-2 pl-1">SUBMIT A PROPOSAL</span>
</q-btn>
```

**Platform (please complete the following information):**
Operating System - Darwin(19.4.0) - darwin/x64
NodeJs - 10.21.0

Global packages
  NPM - 6.14.4
  yarn - 1.22.5
  @quasar/cli - 1.1.0
  cordova - 10.0.0

Important local packages
  quasar - 1.12.8 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 1.9.6 -- Quasar Framework local CLI
  @quasar/extras - 1.8.2 -- Quasar Framework fonts, icons and animations

**Comment by rstoenescu** [comment]
Hi,

You are applying some CSS classes to the QBtn which break the looks of the button. From the naming scheme, I assume that it is your "text-base". But going further, I have some more comments:

1. You can use the "padding" prop on the QBtn to set your custom padding. You're using "py-2" and "pl-1" -- your own, but I assume they are for padding on y axis and padding left. Just mentioning that Quasar has its own q-p*-* classes (example: q-py-sm)
2. Instead of applying no-shadow, you can use the "unelevated" prop.
3. Instead of using the default slot to embed a SPAN element, you can use the "label" prop.

Bottom line, there's nothing wrong with Quasar but with your custom classes.
